documentation/testing/test-plan.md

```diff
@@ -7,67 +7,82 @@
 |---------|---------|-----------|------------|---------------|
 | XX      | QA Name | Test Lead | 01/01/2021 | First Release |
 
+## Project Overview
+### Purpose
+The purpose of the software is to seamlessly integrate the functionalities of View My Financial Insights (VMFI) and School Financial Benchmarking (SFB). The end goal is to provide users with a unified platform where they can compare school financial data, view school spending, and analyse expenditures in comparison with similar schools. Another objective is to build data pipelines that will read data from raw data files and pass them to pipeline to store them into the schema which will later be used in the system.
```

Comment:
Did you want to include the option to view their forecasted returns and ability to model their financial data against custom school comparators?